core/modules/color/color.module

```diff
@@ -581,6 +581,42 @@ function color_save_configuration($theme_name, $theme_info, $palette) {
   );
 }
 
+/**
+ * Implements hook_admin_bar_cache_info().
+ */
+function color_admin_bar_cache_info() {
+  global $theme;
+  $cache = array();
+  $theme_config_name = $theme . '.settings';
+  $theme_config = config($theme_config_name);
+  if ($theme_config && $color = $theme_config->get('color')) {
+    $cache['color'] = array(
```

Comment:
The other implementations of this hook return a `$caches` array (plural). Perhaps that's why this one is not shown under the "Clear caches" menu in the admin bar(?).

Reply:
The link only appears if there are color schemes saved. I changed the color scheme now, it appears now.